boards/arm/nrf5340dk_nrf5340/doc/index.rst

```diff
@@ -54,7 +54,9 @@ contains the processor's information and the datasheet.
 
 .. note::
 
-   In previous Zephyr releases this board was named *nrf5340_dk_nrf5340*.
+   * nRF5340 PDK board is deprecated and will be replaced by nRF5340 DK.
```

Comment:
No need for a bullet list here, but doesn't hurt either.

Reply:
Agreed, this was copy-paste mistake. Will remove ):)

Reply:
removed.